HabitRPG/TableviewCells/Challenges/ChallengeCreatorTableViewCell.swift

```diff
@@ -24,11 +24,25 @@ class ChallengeCreatorTableViewCell: UITableViewCell, ChallengeConfigurable {
     }
     
     func configure(with challenge: Challenge) {
-        configure(user: challenge.user)
+        userNameLabel.text = challenge.leaderName
+        
+        User.fetch(withId: challenge.leaderId) { (user) in
+            self.configure(user: user)
+        }
     }
     
     func configure(user: User?) {
-        userNameLabel.text = user?.username
+        if let member = user {
+            if let avatar = member.getAvatarViewShowsBackground(true, showsMount: false, showsPet: false) {
+                avatarView.addSubview(avatar)
```

Comment:
I might be missing something but doesn't this add more and more avatar views every time the cell is rendered?